cli_reference/developer_cli_odo/creating-an-application-with-a-database.adoc

```diff
@@ -12,7 +12,7 @@ This example describes how to deploy and connect a database to a front-end appli
 * `{odo-title}` is installed.
 * `oc` client is installed.
 * You have a running cluster. Developers can use link:https://cloud.redhat.com/openshift/install/crc/installer-provisioned?intcmp=7013a000002CtetAAC[CodeReady Containers (CRC)] to deploy a local cluster quickly.
-* xref:../../applications/service_brokers/installing-service-catalog.adoc#sb-installing-service-catalog[Service Catalog] is enabled.
+* Service Catalog is enabled.
```

Comment:
Is there a way to determine if Service Catalog is enabled?
Is there a way to turn it off?

Reply:
Chatting with Yana separately to follow-up on what needs to happen here. Was just removing the xref to make the build happy.